tree/ntuple/v7/inc/ROOT/REntry.hxx

```diff
@@ -96,7 +96,7 @@ private:
       fValues.emplace_back(std::move(value));
    }
 
-   /// While building the entry, adds a new value to the list and return the value's shared pointer
+   /// While building the entry, adds a new value to the list of top level fields and returns the value's shared pointer
```

Comment:
I would suggest
```suggestion
   /// While building the entry, adds a new value for the top level field and returns the value's shared pointer
```
The `REntry` doesn't manage the top level fields itself, it has a list of values that correspond to the model's top level fields.